GreenhouseNode/GreenhouseNode/controllers/humidity.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * humidity.h
+ *
+ * Created: 12/6/2021 1:57:19 PM
+ *  Author: tolde
+ */ 
+
+#include <stdio.h>
+#include <time.h>
+#include <stddef.h>
+#ifndef HUMIDITY_H_
```

Comment:
You can also use `#pragma once` instead of all the define things, but it works the same so it doesn't really matter much.